src/desktop/apps/partnerships/client/router.coffee

```diff
@@ -5,8 +5,6 @@ Jump = require '../../../components/jump/view.coffee'
 
 module.exports = class PartnershipsRouter extends Backbone.Router
   routes:
-    'gallery-partnerships': 'toTop'
-    'gallery-partnerships/:slug': 'toSection'
     'institution-partnerships': 'toTop'
     'institution-partnerships/:slug': 'toSection'
     'auction-partnerships': 'toTop'
```

Comment:
I know you've asked if those other pages are still used and the reply was that 'institutions' get some traffic. Wondering if 'auction-partnerships' is relevant to anything? Seems like we've covered most auction houses by now:)

But I guess it's ok to keep for now. -600 lines of code is better than nothing:)

Reply:
yeah, apparently it is still in use - see Jacqueline's comment [here](https://artsy.slack.com/archives/C02MU3DB3/p1598466516020900) and the thread just below that for a bit more detail if you're interested. Sounds like they're planning to update the photos and copy on both of those pages